.github/CONTRIBUTING.md

```diff
@@ -46,25 +49,38 @@ There are different types of component: **AI**, **Data**, **Application**, **Ope
 
 #### AI
 
-**AI** components play a crucial role in transforming unstructured data into formats that are easy to interpret and analyze, thereby facilitating the extraction of valuable insights. These components integrate with AI models from various providers, whether it's the primary Instill Model or those from third-party AI vendors. They are defined and initialized in the [ai](../ai) package.
+**AI** components play a crucial role in transforming unstructured data into
+formats that are easy to interpret and analyze, thereby facilitating the
+extraction of valuable insights. These components integrate with AI models from
+various providers, whether it's the primary Instill Model or those from
+third-party AI vendors. They are defined and initialized in the [ai](../ai)
+package.
 
 #### Data
 
-**Data** components play a crucial role in establishing connections with remote data sources, such as IoT devices (e.g., IP cameras), cloud storage services (e.g., GCP Cloud Storage, AWS S3), data warehouses, or vector databases (e.g., Pinecone). These connectors act as the bridge between VDP and various external data sources. Their primary function is to enable seamless data exchange, enhancing Instill VDP's capability to work with diverse data sources effectively. They are defined and initialized in the [data](../data) package.
+**Data** components play a crucial role in establishing connections with remote
+data sources, such as IoT devices (e.g., IP cameras), cloud storage services
+(e.g., GCP Cloud Storage, AWS S3), data warehouses, or vector databases (e.g.,
+Pinecone). These connectors act as the bridge between VDP and various external
+data sources. Their primary function is to enable seamless data exchange,
+enhancing Instill VDP's capability to work with diverse data sources
+effectively. They are defined and initialized in the [data](../data) package.
```

Comment:
I think the rendered output is a bit cumbersome for these package references, especially the AI package.
![CleanShot 2024-06-03 at 11 16 36](https://github.com/instill-ai/component/assets/3977183/3c96fe87-510d-4af5-b61c-c5e2a05aec40)
![CleanShot 2024-06-03 at 11 18 57](https://github.com/instill-ai/component/assets/3977183/1b0d4b9b-66a5-4a41-99e9-205501855c69)
```suggestion
effectively. They are defined and initialized in the [`data`](../data) package.
```